modules/navController/src/main.cpp

```diff
@@ -140,10 +150,30 @@ class Navigator : public RFModule, public navController_IDL {
     distance_hysteresis_low = abs(rf.check("distance-hysteresis-low", Value(0.2)).asFloat64());
     distance_hysteresis_high = abs(rf.check("distance-hysteresis-high", Value(0.3)).asFloat64());
     base_stop_threshold = rf.check("base-stop-threshold", Value(0.1)).asFloat64();
+    // Offline mode parameter. If 1, this->offline_mode will become true.
+    if(rf.check("offline_mode"))
+    {
+      offline_mode = rf.find("offline_mode").asInt32() == 1;
+      // Force odometry reset parameter. If 1 and if this->offline_mode,
+      // this->force_odometry_reset will become true.
+      if(rf.check("force_odom_reset"))
+      {
+        force_odometry_reset = (rf.find("force_odom_reset").asInt32() == 1 && offline_mode);
+      }
+      // No odom data parameter. If 1 and if this->offline_mode,
+      // this->no_odometry_data will become true.
+      if(rf.check("no_odom_data"))
+      {
+        no_odometry_data = (rf.find("no_odom_data").asInt32() == 1 && offline_mode);
+      }
+    }
```

Comment:
Missing documentation in `navController.xml`.

Reply:
Added documentation